build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -u
+set -e
+set -o pipefail
+
+BROWSERIFY_PATH="./node_modules/browserify/bin/cmd.js"
+BROWSERIFY_CMD="node ${BROWSERIFY_PATH} -s detectEthereumProvider"
```

Comment:
I would expect `browserify` to still work, because this script is run as an npm script. The `$PATH` should be guaranteed to include `node_modules/.bin`, which has `browserify` in it

Reply:
```suggestion
BROWSERIFY_CMD="browserify -s detectEthereumProvider"
```

Reply:
TIL!

Reply:
Oh and as a bonus, this retains compatibility with build systems that don't use a traditional dependency structure (e.g. Yarn v2, tink, etc.)